library/coverage/result_spec.rb

```diff
@@ -1,33 +1,95 @@
 require File.expand_path('../../../spec_helper', __FILE__)
 require 'coverage'
 
+# Clear old results from the result hash
+# https://bugs.ruby-lang.org/issues/12220
+def filtered_result
+  Coverage.result.select { |_k, v| v.any? }
+end
```

Comment:
This helper should be in `library/coverage/fixtures/classes.rb` so it is namespaced and shared, similar to `core/kernel/fixtures/classes.rb`.